numpy/testing/tests/test_utils.py

```diff
@@ -144,7 +145,10 @@ def test_recarrays(self):
         c['floupipi'] = a['floupi'].copy()
         c['floupa'] = a['floupa'].copy()
 
-        self._test_not_equal(c, b)
+        with suppress_warnings() as sup:
```

Comment:
It might cause problems, but maybe its also good, since maybe some project might run into the future warning and did not notice because of this....